maps-compose/src/main/java/com/google/maps/android/compose/GoogleMap.kt

```diff
@@ -150,6 +150,7 @@ public fun GoogleMap(
             }
         }
 
+        // Use [CoroutineStart.UNDISPATCHED] to kick off composition immediately
```

Comment:
The advantage I see is actually more about kicking off GoogleMap loading right away, because that will take place mostly in the background.

Composition happens on main thread, so it will compete with other things, I don't see a big win there.

Reply:
Yes, that's true